src/tests/multihost/ad/test_adparameters_ported.py

```diff
@@ -0,0 +1,1749 @@
+            raiseonerr=False
+        )
+        shortname = hostname_cmd.stdout_text.rstrip()
+        adjoin(membersw='adcli')
+        # Create AD user
+        (aduser, _) = create_aduser_group
+        # Configure sssd
+        ad_realm = multihost.ad[0].domainname.upper()
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'debug_level': '9',
+            'ad_domain': multihost.ad[0].domainname.lower(),
+            'ad_server': multihost.ad[0].hostname,
+            'use_fully_qualified_names': 'False',
+            'cache_credentials': 'True',
+            #'ldap_id_mapping': 'False',
+            'ldap_sasl_authid': f'host/{shortname}.{ad_domain}@{ad_realm}',
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        client.clear_sssd_cache()
+        # Search for the user
+        usr_cmd = multihost.client[0].run_command(
+            f'getent passwd {aduser}',
+            raiseonerr=False
+        )
+        # Run su command
+        su_cmd = multihost.client[0].run_command(
+            f'su - {aduser} -c  whoami',
+            raiseonerr=False
+        )
+        # Download the sssd domain log
+        log_str = multihost.client[0].get_file_contents(
+            f"/var/log/sssd/sssd_{multihost.ad[0].domainname.lower()}.log"). \
+            decode('utf-8')
+
+        # TODO: DELETE
+        multihost.client[0].run_command(
+            f"cat /var/log/sssd/sssd_{multihost.ad[0].domainname.lower()}.log",
+            raiseonerr=False
+        )
+
+        client.restore_sssd_conf()
+        client.clear_sssd_cache()
+
+        assert f"Option ldap_sasl_authid has value host/{shortname}.{ad_domain}@{ad_realm}" in log_str
+        assert f"authid contains realm" in log_str
+        assert f"Will look for host/{shortname}.{ad_domain}@{ad_realm} in" in log_str
+        assert f"Trying to find principal host/{shortname}.{ad_domain}@{ad_realm} in keytab" in log_str
+        assert f"Principal matched to the sample (host/{shortname}.{ad_domain}@{ad_realm})" in log_str
+        assert usr_cmd.returncode == 0, f"User {aduser} was not found!"
+        assert su_cmd.returncode == 0, f"Su for user {aduser} failed!"
+
+    @staticmethod
+    @pytest.mark.broken
+    def test_0010_ad_parameters_ldap_sasl_short(multihost, adjoin,
+                                                create_aduser_group):
+        """
+        :title: IDM-SSSD-TC: ad_provider: ad_parameters: Using short principal
+        :id: 6f1cc204-0dd3-40eb-a3e2-a113cc7c2df3
+        :setup:
+         1. Configure ,
+            clear cache and restart sssd.
+         2. Create an AD user.
+        :steps:
+          1. Run getent passwd for the user and verify the home location.
+        :expectedresults:
+          1. User is found and homedir is overridden.
```

Comment:
This isn't related to the test?

The only difference should be the REALM.COM is omitted from the ldap_sasl_authid, so host/fully.qualified.hostname.com instead of host/fully.qualified.hostname.com@REALM.COM

Reply:
C&P mistake, I did not manage to get the sasl tests working yet so I did not take spend time on documentation.
I will make sure to fix it when the test are working.